ambari/security/kerberos/kdc-config.rst

```diff
@@ -0,0 +1,14 @@
+Installing and Configuring the KDC
+==================================
+
+Use an Exisiting MIT KDC
+________________________
+
+Use an Existing Active Directory
+________________________________
+
+Use Manual Kerberos Setup
+_________________________
+
+(Optional) Install a new MIT KDC
```

Comment:
Maybe we don't want to provide assistance on KDC setup. I think that this part should be limited to configuring existing KDC only.